client/electron/electron-builder.json

```diff
@@ -37,11 +37,9 @@
     "maintainer": "Jigsaw LLC",
     "target": [{
       "arch": [
-        "x64"
+        "x64",
+        "arm64"
```

Comment:
This produces two binaries: `Outline-Client-arm64.deb` and `Outline-Client-amd64.deb`. But we only build the ARM64 version of the tun2socks (aka the one built by `client/go/Taskfile.yml`), so it will break the x64 version `Outline-Client-amd64.deb`.

I would suggest to build both architectures in `client/go/Taskfile.yml`, maybe using separate output folders. But then it's tricky to include different binaries for different architecture in `electron-builder.json`.

Alternatively, we should limit electron to only build one single architecture, but then you need to pass the CLI argument to `electron-builder` now, which is also not straightforward.